autotst/conformer/systematic.py

```diff
@@ -224,23 +178,59 @@ def opt_conf(i, rmsd_cutoff):
             except RuntimeError:
                 logging.info("Optimization failed...we will use the unconverged geometry")
                 pass
-
+        conf_copy = conformer.copy()
         conformer.update_coords_from("ase")  
-        energy = get_energy(conformer)
+        energy = conformer.ase_molecule.get_potential_energy()
         conformer.energy = energy
-        if len(return_dict)>0:
-            conformer_copy = conformer.copy()
-            for index,post in return_dict.items():
-                conf_copy = conformer.copy()
-                conf_copy.ase_molecule.positions = post
-                conf_copy.update_coords_from("ase")
-                rmsd = rdkit.Chem.rdMolAlign.GetBestRMS(conformer_copy.rdkit_molecule,conf_copy.rdkit_molecule)
-                if rmsd <= rmsd_cutoff:
-                    return True
-        if str(i) != 'ref':
-            return_dict[i] = conformer.ase_molecule.get_positions()
-        return True
+        
+        return conformer
+def systematic_search(conformer,
+                      delta=float(120),
+                      energy_cutoff = 10.0, #kcal/mol
+                      rmsd_cutoff = 0.5, #angstroms
+                      cistrans = True,
+                      chiral_centers = True,
+                      multiplicity = False,
+                      ):
+    """
+    Perfoms a systematic conformer analysis of a `Conformer` or a `TS` object
 
+    Variables:
+    - conformer (`Conformer` or `TS`): a `Conformer` or `TS` object of interest
+    - delta (int or float): a number between 0 and 180 or how many conformers to generate per dihedral
+    - cistrans (bool): indication of if one wants to consider cistrans bonds
+    - chiral_centers (bool): indication of if one wants to consider chiral centers bonds
```

Comment:
Should add `rmsd_cutoff` and `energy_cutoff` to the doc string. Also add that these can be strings because it's allowed in the code but not specified here.